docs/subnets/consensus-based-weights.md

```diff
@@ -16,23 +16,23 @@ With this feature, the dividends to a subnet validator are better correlated to
 
 ## Collab notebooks
 
-A subnet owner can run the `weight_copy/liquid_alpha_diagnostic.ipynb` in the below Python notebook to experiment and choose the right values for the hyperparameters `alpha_low`, `alpha_high`, `commit_reveal_interval`.
+A subnet owner can run the `weight_copy/liquid_alpha_diagnostic.ipynb` in the Python notebook below to experiment and choose the right values for the hyperparameters `alpha_low`, `alpha_high`, and `commit_reveal_interval`.
 
 - For commit reveal diagnostic: https://colab.research.google.com/github/opentensor/developer-docs/blob/main/static/weight_copy/commit_reveal_diagnostic.ipynb?authuser=5
 - For liquid alpha diagnostic: https://colab.research.google.com/github/opentensor/developer-docs/blob/main/static/weight_copy/liquid_alpha_diagnostic.ipynb?authuser=5
 - [GitHub directory with Python notebooks](https://github.com/opentensor/developer-docs/tree/main/static/weight_copy/). 
 
 ## Description
 
-Currently while calculating the dividends to a subnet validator, a quantity called $\Delta B_{ij}$, defined as an **instantaneous bond value** of a subnet validator $i$ with the subnet miner $j$, is used.
+Currently, while calculating the dividends to a subnet validator, a quantity called $\Delta B_{ij}$, defined as an **instantaneous bond value** of a subnet validator $i$ with the subnet miner $j$, is used.
 
-A subnet validator maintains an instantaneous bond value on each subnet miner. However, while calculating a subnet validator's dividends, instead of directly using the instantaneous bond value $\Delta B_{ij}$, we use $B_{ij}^{(t)}$, an **exponential moving average (EMA)** of the bond value, weighted over the current epoch and the previous epoch. See the below equation for how this EMA is computed, where $t$ is the current epoch time and $t-1$ is the previous epoch time:
+A subnet validator maintains an instantaneous bond value on each subnet miner. However, while calculating a subnet validator's dividends, instead of directly using the instantaneous bond value $\Delta B_{ij}$, we use $B_{ij}^{(t)}$, an **exponential moving average (EMA)** of the bond value, weighted over the current epoch and the previous epoch. See the below equation for how this EMA is computed, where $t$ is the current epoch time and $t-1$ is the last time epoch:
```

Comment:
Let's keep the "previous epoch". This is because "last" in English can mean "final" and we don't want that confusion. Also when we say "t" is current, then "t-1" is previous.

Reply:
Done